packages/sources/cryptocompare-test/src/endpoint/crypto-ws.ts

```diff
@@ -61,17 +61,33 @@ export const wsTransport = new WebSocketTransport<WsEndpointTypes>({
       if (message.TYPE === '5' && 'PRICE' in message) {
         return [
           {
-            params: { base: message.FROMSYMBOL, quote: message.TOSYMBOL, endpoint: 'crypto-ws' },
-            value: message.PRICE as number,
+            params: { base: message.FROMSYMBOL, quote: message.TOSYMBOL, endpoint: 'crypto' },
```

Comment:
Assuming we're (for now) only routing based on the env var, we should be able to remove "endpoint" here and from the input params (`cryptoEndpointInputParams`).

Reply:
Actually i think it makes sense to have the endpoint explicitly in the params.  That way we don't need to have separate endpoint/transport for `marketcap` and `volume` since we are using `endpointResultPaths`.  What do you think ?

Reply:
Hmm yeah good point. I think we can just proceed with how it is now then!